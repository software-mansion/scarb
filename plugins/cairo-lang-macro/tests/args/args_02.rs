use cairo_lang_macro::{attribute_macro, ProcMacroResult, TokenStream};

#[attribute_macro(parent = "a-b")]
fn t1(_a: TokenStream, _b: TokenStream) -> ProcMacroResult {
    ProcMacroResult::new(TokenStream::empty())
}

fn main() {}
