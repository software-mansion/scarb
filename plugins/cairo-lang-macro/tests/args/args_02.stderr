error: `parent` argument is not a valid path
 --> tests/args/args_02.rs:3:28
  |
3 | #[attribute_macro(parent = "a-b")]
  |                            ^^^^^

warning: unused imports: `ProcMacroResult` and `TokenStream`
 --> tests/args/args_02.rs:1:41
  |
1 | use cairo_lang_macro::{attribute_macro, ProcMacroResult, TokenStream};
  |                                         ^^^^^^^^^^^^^^^  ^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
